feat: Added published date to versioning changelist (#83)
FidelityInternational/djangocms-pageadmin: CHANGELOG.rst

Unreleased
==========

> * feat: Replacing created with published date for the versioning changelist

Author: ```suggestion
* feat: Replacing created with published date in versioning changelist
```
